sktime/markov/hmm/init/gaussian/_init_gaussian_impl.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+
+
+def from_data(dtrajs, n_hidden_states, reversible):
+    r""" Makes an initial guess :class:`HMM <HiddenMarkovStateModel>` with Gaussian output model.
+
+    To this end, a Gaussian mixture model is estimated using `scikit-learn <https://scikit-learn.org/>`_.
+
+    Parameters
+    ----------
+    dtrajs : array_like or list of array_like
+        Trajectories which are used for making the initial guess.
+    n_hidden_states : int
+        Number of hidden states.
+    reversible : bool
+        Whether the hidden transition matrix is estimated so that it is reversible.
```

Comment:
this implies there is only one way to enforce reversibility right?

Reply:
it is just about the reversibility of the initial guess hmm, you can still have non-reversible hmm estimate afterwards in the EM algorithm